pkgs/by-name/so/sonarr/package.nix

```diff
@@ -96,7 +117,7 @@ buildDotnetModule {
   ];
 
   dotnetFlags = [
-    "--property:TargetFramework=net6.0"
+    "--property:TargetFramework=net8.0"
     "--property:EnableAnalyzers=false"
     # Override defaults in src/Directory.Build.props that use current time.
     "--property:Copyright=Copyright 2014-2024 sonarr.tv (GNU General Public v3)"
```

Comment:
Can you please update the year to 2025?

This is something that realistically has to be done at once per year so not worth the effort of automating via updateScript 😅

Reply:
Done